functions/Invoke-DbaDbDataMasking.ps1

```diff
@@ -71,6 +71,14 @@ function Invoke-DbaDbDataMasking {
     .PARAMETER CommandTimeout
         Timeout for the database connection in seconds. Default is 300.
 
+    .PARAMETER DictionaryFilePath
+        Import the dictionary to be used in in the database masking
+
+    .PARAMETER DictionaryExportPath
```

Comment:
Should this be an Export command instead?

Reply:
The functionality to export the dictionary will, and can, only be used in the Invoke-DbaDbDataMasking command.